src/components/DNSMap/DNSMap.tsx

```diff
@@ -110,15 +110,15 @@ const DNSMap = () => {
         const args = [`${values.domain}`, "-d", `${values.delay}`];
 
         if (values.wordlistPath) {
-            args.push(`-S ${values.wordlistPath}`);
+            args.push(`-w`, `${values.wordlistPath}`);
```

Comment:
This code work fine but i don't see why you would need to wrap your `values.wordlistPath` in a `''` as in the form the value is already define as a string. Same goes with the other if statement.